### PR TITLE
DHFPROD-2356: Quickstart UI Automation on jobs page validation

### PR DESCRIPTION
**Pull request by timur-isangulov**
QS Jobs page validation tests.

### CODE REVIEW

File: web/e2e/page-objects/jobs/jobs.ts [hunk 1]
```diff
@@ -148,6 +156,64 @@ export class JobsPage extends AppPage {
   jobOutputCloseButton() {
     return element(by.css('app-job-output mdl-button'));
   }
+
+
+  resetFiltersButton() {
```
Comment (ayuwono) [inline web/e2e/page-objects/jobs/jobs.ts]: some of these page objects like filtering are already exist on manageJobs.ts. Can you just delete jobs.ts and use manageJobs.ts, so we are consistent with manageFlows structure. Also, you can re-use the page objects on manageJobs.ts, we use these page objects on scenario tests. If you make some changes, make sure that it's not breaking the scenarios tests

File: web/e2e/page-objects/appPage.ts [hunk 2]
```diff
@@ -92,6 +92,14 @@ export class AppPage extends Page {
     await browser.waitForAngular();
   }
 
+  async clickJobsTab() {
+    return await browser.executeScript("arguments[0].click();", this.jobsTab);
+  }
```
Comment (ayuwono) [inline web/e2e/page-objects/appPage.ts]: add await browser.waitForAngular() too here?

Comment (timur-isangulov) [reply web/e2e/page-objects/appPage.ts]: Added. Thanks.

File: web/e2e/page-objects/jobs/jobs.ts (hunk 1 above)
Comment (timur-isangulov) [reply web/e2e/page-objects/jobs/jobs.ts]: I moved methods to manageJobs.ts page object, will review and remove unused classes later.

**Pull request merged**